debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+VERSION=0.1
+DEB_DIR=debian/streamgraph$(VERSION)
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+
+override_dh_auto_build:
+	dh_auto_build
+
+override_dh_auto_install:
+	mkdir -p $(DEB_DIR)/usr/bin
+	install -m 755 obj-x86_64-linux-gnu/bin/streamgraph $(DEB_DIR)/usr/bin/streamgraph$(VERSION)
```

Comment:
This could also go in a [install][1] files.

[1]: https://www.debian.org/doc/manuals/maint-guide/dother.en.html#install